test/run/ch-run_misc.bats

```diff
@@ -1009,3 +1009,30 @@ EOF
     [[ $output = *"UID ${uid_bad} not found; using dummy info"* ]]
     [[ $output = *"GID ${gid_bad} not found; using dummy info"* ]]
 }
+
+@test 'Docker metadata' {
+    scope standard
+    [[ $CH_BUILDER = docker ]] || skip 'Docker only'
+    image="centos7"
```

Comment:
Would [`charliecloud/metadata`](https://hub.docker.com/r/charliecloud/metadata) work? That would give a more complete test.

Reply:
Yeah that sounds good!